src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/personalcareneeds_offender_validation.json

```diff
@@ -1,5 +1,5 @@
 {
   "status": 400,
-  "userMessage": "getPersonalCareNeeds.offenderNo: offenderNo: must not be empty",
-  "developerMessage": "getPersonalCareNeeds.offenderNo: offenderNo: must not be empty"
+  "userMessage": "getPersonalCareNeeds.offenderNos: offenderNo: must not be empty",
```

Comment:
How come this has changed?

Reply:
Interface was different to implementation in variable name